src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/IepLevelService.kt

```diff
@@ -2,35 +2,24 @@ package uk.gov.justice.digital.hmpps.incentivesapi.service
 
 import org.springframework.stereotype.Service
 import uk.gov.justice.digital.hmpps.incentivesapi.config.DataIntegrityException
-import uk.gov.justice.digital.hmpps.incentivesapi.dto.prisonapi.IepLevel
+import uk.gov.justice.digital.hmpps.incentivesapi.dto.PrisonIncentiveLevel
 
 @Service
 class IepLevelService(
   private val incentiveLevelService: IncentiveLevelService,
-  private val prisonIncentiveLevelService: PrisonIncentiveLevelService,
 ) {
-
-  suspend fun getIepLevelsForPrison(prisonId: String): List<IepLevel> {
-    var i = 1
-    return prisonIncentiveLevelService.getActivePrisonIncentiveLevels(prisonId)
-      .map {
-        IepLevel(
-          iepLevel = it.levelCode,
-          iepDescription = it.levelName,
-          sequence = i++,
-          default = it.defaultOnAdmission,
-          active = it.active,
-        )
-      }
-  }
-
-  fun chooseDefaultLevel(prisonId: String, prisonLevels: List<IepLevel>): String {
-    val activePrisonLevels = prisonLevels.filter(IepLevel::active)
-    return activePrisonLevels.firstOrNull(IepLevel::default)?.iepLevel
+  /**
+   * Selects the default level (for new admissions) for a prison
+   * NB: this existed largely for compatibility with NOMIS which allowed there to be no default set,
+   * business rules now prevent this from happening
+   */
+  fun chooseDefaultLevel(prisonId: String, prisonLevels: List<PrisonIncentiveLevel>): String {
+    val activePrisonLevels = prisonLevels.filter(PrisonIncentiveLevel::active)
```

Comment:
now, the whole method could probably be removed as there's _always_ a default. i was being a bit conservative with this PR because the data wasn't fixed in production until after this morning's go-live

Reply:
Yeah that's what I'm thinking about the first part 🤔 

I think what you did is fine, but if you fancy I think `chooseDefaultLevel()` could be possibly removed/made private or at least simplified.

Similarly the class could use `PrisonIncentiveLevelService` directly and don't have to rely on client code to pass the list of levels (the client code currently use this service to get the list of prison levels).